publish.sh

```diff
@@ -184,7 +184,12 @@ fi
 
 RELEASE_VERSION_FAMILY=$(echo "$RELEASE_VERSION" | sed -E 's/^([0-9]+\.[0-9]+).*/\1/')
 
+IS_GRADLE_PROJECT=0
 if [ "$PROJECT" == "orm" ] || [ "$PROJECT" == "reactive" ] || [ "$PROJECT" == "models" ]; then
+  IS_GRADLE_PROJECT=1
+fi
```

Comment:
Could probably be autodetected, like we do in e.g. `update-version.sh`?

```suggestion
if [ -f "./gradlew"]; then
	IS_GRADLE_PROJECT=1
else
	IS_GRADLE_PROJECT=0
fi
```

It doesn't matter for the main projects where an explicit list is fine, but it could for side-projects (infra, ...).


Though at this point you can likely inline the condition :)